docs/state-store-overview.md

```diff
@@ -34,6 +42,37 @@ Streaming state store has distinguished workload characteristics.
 
 This leads to the design of Hummock, the cloud-native KV-based streaming state store. We’ll explain concepts like “epoch”, “key space” and “barrier” in the following chapters.
 
+## Relational Table
+[source code](https://github.com/singularity-data/risingwave/blob/main/src/storage/src/table/state_table.rs)
+
+In this part, we will introduce how stateful executors interact with KV state store through the relational table layer.
+
+Relational table layer consists of State Table, Mem Table and Cell-based Table. The State Table provides the table operations by these APIs: `get_row`, `scan`, `insert_row`, `delete_row` and `update_row`, which are the read and write  interfaces for executors. The Mem Table is an in-memory buffer for caching table operations during one epoch, and the Cell-based Table is responsible for performing serialization and deserialization between cell-based encoding and KV encoding.
+
+![Overview of Relational Table](images/relational-table-layer/relational-table-01.svg)
+
+### Write Path
+Executors perform operations on relational table, and these operations will first be cached in Mem Table. Once an executor wants to write these operations to state store, cell-based table will covert these operations into kv pairs and write to state store with specific epoch. 
```

Comment:
Give some examples. What will happen when, for example, join executors writes some state into the relational table? And what will happen when commit?

Reply:
> 

Good idea!